src/radix_connect/mobile/relay_service/service.rs

```diff
@@ -127,91 +93,12 @@ impl Service {
-        let mock_antenna_with_spy =
-            MockAntenna::with_spy(200, vec![], |request| {
-                let relay_request =
-                    Request::new_get_requests(Session::sample().id);
-                let encoded_request =
-                    serde_json::to_vec(&relay_request).unwrap();
-
-                let expected_request = NetworkRequest {
-                    url: Url::from_str(SERVICE_PATH).unwrap(),
-                    method: NetworkMethod::Post,
-                    body: encoded_request.into(),
-                    headers: HashMap::new(),
-                };
-
-                pretty_assertions::assert_eq!(
-                    request.url,
-                    expected_request.url
-                );
-                pretty_assertions::assert_eq!(
-                    request.method,
-                    expected_request.method
-                );
-                pretty_assertions::assert_eq!(
-                    request.body,
-                    expected_request.body
-                );
-            });
-
-        let service =
-            Service::new_with_network_antenna(Arc::new(mock_antenna_with_spy));
-        let session = Session::sample();
-
-        let req = service.get_wallet_interaction_requests(session);
-        let _ = timeout(MAX, req).await.unwrap();
-    }
-
-    #[actix_rt::test]
-    async fn test_get_wallet_interaction_requests_failure() {
-        let service = Service::new_always_failing();
-        let session = Session::sample();
-
-        let req = service.get_wallet_interaction_requests(session);
-        let result = timeout(MAX, req).await.unwrap();
-        assert!(result.is_err());
-    }
-
-    #[actix_rt::test]
-    async fn test_get_wallet_interaction_requests() {
-        let session = Session::sample();
-        // Prepare encryption keys
-        let mut encryption_key = session.encryption_key;
-
-        // Serialize the request
-        let dapp_to_wallet_interaction_unvalidated =
-            DappToWalletInteractionUnvalidated::sample();
-        let body = dapp_to_wallet_interaction_unvalidated_to_json_bytes(
-            &dapp_to_wallet_interaction_unvalidated,
-        );
-
-        // Encrypt the request
-        let encrypted = EncryptionScheme::default()
-            .encrypt(body.to_vec(), &mut encryption_key);
-        let requests = vec![encrypted];
-        let encoded_requests = serde_json::to_vec(&requests).unwrap();
-
-        // Setup Service
-        let service = Service::new_succeeding_http_client(encoded_requests);
-
-        // Get interaction requests
-        let req = service.get_wallet_interaction_requests(session);
-        let result = timeout(MAX, req).await.unwrap();
-
-        // Assert the result
-        pretty_assertions::assert_eq!(
-            result,
-            Ok(vec![dapp_to_wallet_interaction_unvalidated])
-        );
-    }
+    const SERVICE_PATH: &str =
```

Comment:
Remove this duplication  of `https://radix-connect-relay.radixdlt.com/api/v1`?, it shadows the outer `SERVICE_PATH`! for which we wanna write a unit test :)